demisto_sdk/commands/common/errors.py

```diff
@@ -278,6 +278,9 @@
     "invalid_file_path_layout": {'code': "LO102", 'ui_applicable': False, 'related_field': ''},
     "invalid_file_path_layoutscontainer": {'code': "LO103", 'ui_applicable': False, 'related_field': ''},
     "invalid_incident_field_in_layout": {'code': "LO104", 'ui_applicable': False, 'related_field': ''},
+    "invalid_file_path_pre_process_rules": {'code': "PP100", 'ui_applicable': False, 'related_field': ''},
```

Comment:
remove, and update the error codes of the other errors

Reply:
https://github.com/demisto/demisto-sdk/commit/54232468026fd952d6ca3139e65581848e09c4e5